src/reusableComponents/Logo/index.jsx

```diff
@@ -8,7 +8,7 @@ const Logo = ({ ml }) => {
   return (
     <p
       id="dummygram-logo"
-      style={{ marginLeft: ml ? ml : "20px" }}
+      style={{ display:"flex", margin:".5vh auto", marginBottom:"2vh"}}
```

Comment:
you're modifying a reusable component but still not checking the sideeffect

Reply:
nor handling the props

Reply:
in fact, the logo is still not in center of login form, and the logo is misbehaving in other parts of the website as well due to this